.schema/0.8.schema.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/jetpack-io/devbox",
+    "title": "Devbox json definition",
+    "description": "Defines fields and acceptable values of devbox.json",
+    "type": "object",
+    "properties": {
+        "packages": {
+            "description": "A specification of packages to be installed via devbox.",
+            "oneOf": [
+                {
+                    "type": "array",
+                    "items": {
+                        "description": "Name and version of each package in name@version format.",
+                        "type": "string"
+                    }
+                },
+                {
+                    "type": "object",
+                    "description": "Name of each package in {\"name\": \"version\"} format.",
+                    "patternProperties": {
+                        ".*": {
+                            "type": "string",
+                            "description": "Version number of the package in this object."
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "description": "Name of each package in {\"name\": {\"version\": \"1.2.3\"}} format.",
+                    "patternProperties": {
+                        ".*": {
+                            "type": "object",
+                            "description": "Version number of the specified package in {\"version\": \"1.2.3\"} formmat.",
```

Comment:
This object can contain a few more fields. I think they are:

- version
- platforms - only install the package on these Nix systems (e.g., `aarch64-darwin`, `x86_64-linux`, etc.)
- excluded_platforms - don't install the package on these Nix systems.
- glibc_patch - apply the glibc-patch feature to this package's binaries.

Reply:
@gcurtis can you send me an example of devbox.json with these parameters included? 
Also a list of all possible values for platforms/excluded_platforms.
Side note: If I remember correctly, platforms and excluded_platforms can not co-exist right?